packages/cli/src/commands/app/list.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import util from 'util'
+
+import Command from '../../OpenVaultCommand'
+
+export default class AppListCommand extends Command {
+  static description = `List apps you've created and installed`
+  static flags = Command.flags
+
+  async run() {
+    // $FlowFixMe: indexer property
+    const res = await this.client.app.getAll()
+    Object.keys(res).forEach(type => {
+      this.log(`\n${type === 'own' ? 'Created' : 'Installed'} Apps\n`)
```

Comment:
Not an issue but rather than formatting the console output manually like this, you could use the `cli-table` library, as it's used here: https://github.com/MainframeHQ/js-mainframe/blob/master/packages/cli/src/commands/vault/settings.js#L54-L61